### PR TITLE
[WIP] Adding __repr__ for Track objects

### LINKED ISSUE #74: implement __repr__ for pretty printing Track objects
Implement a custom `__repr__` method for each `Track` object to make it more transparent to users what data is contained in each track object.

### PR DESCRIPTION
**Pull request by tkell**
Hello hello!

This PR will fix #74!  I did `__repr__` functions for all the existing datasets, and ran `black`, too – let me know what you think!

cc @rabitt, @andreasjansson

### CODE REVIEW

File: mirdata/beatles.py [hunk 1]
```diff
@@ -49,41 +49,50 @@ class Track(object):
...
         self.audio_path = utils.get_local_path(
-            self._data_home, self._track_paths['audio'][0])
+            self._data_home, self._track_paths["audio"][0]
+        )
 
-        self.title = os.path.basename(
-            self._track_paths['sections'][0]).split('.')[0]
+        self.title = os.path.basename(self._track_paths["sections"][0]).split(".")[0]
+
+    def __repr__(self):
```
Comment (rabitt) [inline mirdata/beatles.py]: Can you print all of the cached properties too, something like `beats=BeatData('start_times', 'end_times', 'sections')`

File: mirdata/beatles.py [hunk 2]
```diff
@@ -22,12 +22,12 @@
 
 import mirdata.utils as utils
 
-DATASET_DIR = 'Beatles'
-INDEX = utils.load_json_index('beatles_index.json')
+DATASET_DIR = "Beatles"
```
[the remark this answers is not in the document]
Comment (tkell) [reply mirdata/beatles.py]: 💯

File: mirdata/beatles.py (hunk 1 above)
Comment (tkell) [reply mirdata/beatles.py]: Can do!

**Comment by tkell** [comment]
@rabitt, re-wrote this to have v. big reprs, as per your request – let me know what you think!

**Comment by tkell** [comment]
Can do!

**Comment by tkell** [comment]
Closing as #94 is an updated version!

**Pull request closed without merging**